README.md

```diff
@@ -2,4 +2,159 @@
+
+### `Card.swift`
+
+1. Re-enable the target membership for `CardSpec.swift` for `swift-blackjackTests`.
+
+2. Create a class file for the `Card` class. It will need the same properties as the `FISCard` class from `objc-blackjack`:
+  * `suit`, a string,
+  * `rank`, a string,
+  * `cardLabel`, a string, and
+  * `cardValue`, an unsigned integer.
+
+3. Write an initializer for `Card` that takes two arguments for `suit` and `rank`.
+  * It should also set the `cardLabel` property to string containing both the suit and the rank in the form of `♠︎A` (for the Ace of Spades).
+  * It should also set the `cardValue` property appropriately for scoring Blackjack: Aces are worth one, number cards are worth their face values, and face cards are worth ten.
+
+4. Make the `Card` class conform to the `CustomDebugStringConvertible` protocol. A `Card` object should print its `cardLabel` string to the console when it is submitted to the `print()` function.
+
+5. Set up two class methods called `validSuits` and `validRanks` that return arrays containing the four suit icons and the thirteen string representations of the ranks (Ace through King).
+
+### `Deck.swift`
+
+1. Re-enable the target membership for `DeckSpec.swift` for `swift-blackjackTests`.
+
+2. Create a class file for the `Deck` class. It should have two private array properties for holding cards: one for holding the remaining (undealt) cards, and the other for holding the dealt cards. The test file cannot see them so name them yourself.
+
+3. Write an initializer that generates the fifty-two unique cards to represent a standard deck of playing cards and holds them in the array of cards that can be dealt.
+
+4. Make the `Deck` class conform to the `CustomDebugStringConvertible` protocol. The deck should print the count of its remaining cards and dealt cards to the console. Look at the tests to see the exact substrings they are expecting.  
+**Top-tip:** *Set this up as a calculated property.*
+
+5. Write the `drawCard()` function to return the next card. It should remove that method from the remaining cards and add it to the dealt cards.
+
+6. Write the `reshuffle()` method to randomize the cards that have not yet been dealt out.  
+**Hint:** *The* `arc4random_uniform()` *C function is accessible in Swift also.*
+
+7. Write the `shuffle()` method that shuffles the dealt cards back into the cards that are ready to deal, randomizing them again.
+
+### `Player.swift`
+
+1. Re-enable the target membership for `PlayerSpec.swift` for `swift-blackjackTests`.
+
+2. Create a new class file for the `Player` class. Have it conform to the `CustomDebugStringConvertible` protocol. It will need nine properties total:
+  * `name`, a string,
+  * `cards`, an array of `Card` objects,
+  * `handscore`, an unsigned integer,
+  * `blackjack`, a boolean,
+  * `busted`, a boolean,
+  * `stayed`, a boolean,
+  * `mayHit`, a boolean,
+  * `wallet`, an unsigned integer, and
```

Comment:
problematic to use floating-point values for currency because of floating-point rounding. I was trying to avoid a direct reference to currency because of gambling anyway, maybe we should call this `tokens`? It was weird scoring blackjack as simple wins & losses in the Objective-C version.

http://stackoverflow.com/questions/28421176/which-swift-datatype-do-i-use-for-currency